x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -296,6 +296,7 @@ interface AllowlistFields {
 // Allow list process fields within events.  This includes "process" and "Target.process".'
 /* eslint-disable @typescript-eslint/naming-convention */
 const allowlistProcessFields: AllowlistFields = {
+  args: true,
   name: true,
   executable: true,
   command_line: true,
```

Comment:
could you also add malware_signature under process.Ext below?

Reply:
added to `dll.Ext` as well
1a5b68a2b77abccfc950dc950d4472795d4592e0